packages/jsii-pacmak/lib/targets/go/types/go-type-reference.ts

```diff
@@ -97,6 +96,6 @@ export class GoTypeRef {
 
     // type isn't handled
     // TODO: Are there other cases to handle? if not throw an error.
```

Comment:
Perhaps drop a warning so you know something unexpected is about to happen?

Reply:
Added a debug message, using `log.warn` makes this annoying the the interim =).